robolectric/src/test/java/org/robolectric/shadows/ShadowNetworkCapabilitiesTest.java

```diff
@@ -57,4 +62,30 @@ public void hasCapability_shouldReturnAsPerAssignedCapabilities() {
     assertThat(networkCapabilities.hasCapability(NET_CAPABILITY_TRUSTED)).isTrue();
     assertThat(networkCapabilities.hasCapability(NET_CAPABILITY_NOT_VPN)).isTrue();
   }
+
+  @Test
+  @Config(minSdk = O)
+  public void getNetworkSpecifier_shouldReturnTheSpecifiedValue_fromO() {
+    NetworkCapabilities networkCapabilities = ShadowNetworkCapabilities.newInstance();
+    // Required to set NetworkSpecifier
+    shadowOf(networkCapabilities).addTransportType(TRANSPORT_WIFI);
+
+    NetworkSpecifier testNetworkSpecifier = new TestNetworkSpecifier("testNetworkSpecifier");
```

Comment:
TestNetworkSpecifier seems to be added in S. Weirdly this test fails when I try to run it in from O (26) to R (30) internally at Google. Very strange, it seems like this test is just not run in GitHub CI.

In the meantime can you make this test `minSdk = S`?

Reply:
Thanks for the correction.
I can just replace it with mock because in the test I just check the equivalence of instances `NetworkSpecifier`.

Reply:
Sounds good to me.

Reply:
Fixed, thank you.